xunit-autolabeler-v2/ast_parser/lib/constants.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+IGNORED_METHOD_NAMES = [
```

Comment:
Avoid using mutable global constants to avoid hard to find bugs. Consider to make this a tuple or a frozenset (not sure how are you planning to use this, right now looks unused).

Reply:
This is exclusively imported into other files.

Considering that, is making this immutable still worth doing? (Happy to make the change if so.)

Reply:
Yes, things like this could happen that are hard to debug:

def process_files(is_special_case):
  methods_to_ignore = IGNORED_METHOD_NAMES
  if is_special_case:
    methods_to_ignore += ['another_method']
  process_and_ignore(methods_to_ignore)

Now your global list has an extra item.

Reply:
Done - thanks for clarifying!